.github/workflows/lint.yml

```diff
@@ -0,0 +1,37 @@
+name: Lint
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  htmlproofer:
+    name: Run htmlproofer
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Ruby
+        uses: actions/setup-ruby@v1
+        with:
+          ruby-version: 2.7.x
+      - uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-
+      - name: Bundle install
+        run: |
+          bundle config path vendor/bundle
+          bundle install
+      - run: bundle exec jekyll build
+      - name: Cache HTMLProofer
```

Comment:
Use the same cache job from above but with multiple paths

Reply:
I think you can do

```yml
path: |
  vendor/bundle
  .tmp/htmlproofer
```

Reply:
https://github.com/ruby/setup-ruby#caching-bundle-install-automatically suggests something else for gem installs... Will leave that and the htmlproofer cache in.

Reply:
Caching looks good: https://github.com/PriceChild/libera-chat.github.io/runs/2788172469?check_suite_focus=true